### DISCUSSION TITLE
[auth] Increase code density

### DISCUSSION THREAD

**Discussion by dnred** [ente-io/ente#4365] [Enhancements]
Add a view to display even more codes and not waste as much space.

| Stratum Auth | Aegis |
|:---------------:|:------:|
| ![Stratum Auth](https://github.com/user-attachments/assets/c05bf1cc-b396-495e-b9b4-b60113548bbb) | <img height="715" alt="Aegis" src="https://github.com/user-attachments/assets/d926b3f9-c35f-48d5-af91-f5aa4356e876" /> |

Related: #1030

**Comment by kaizokan** [comment]
For some it's wasted space for others it's "Less is more" ; ) and for others it could be privacy (showing less of which accounts you own).
I personally like the one item per row in compact mode but that is just me.

I don't know about you and others, but I personally do not use the "Next code" and even have my codes hidden anyway. If such a grid mode is used maybe the "Next code" can be removed to have more space? I don't know how often "Next code" is used, I guess that also depends on whether users copy the code or switch back and forth to enter the code (which can take longer). I personally never use it and codes that have disappeared (but copied or remembered) can still be used based on my experience. Just sharing my thoughts.
